godtools/Views/TractElements/TextContent.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TextContent.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/26/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class TextContent: BaseTractElement {
+    
+    override func setupView(properties: Dictionary<String, Any>) {
+        let attributes = loadElementAttributes(properties: properties)
+        let label = GTLabel(frame: createLabelFrameForHeight(self.height))
+        let text: String = attributes.text
+        let backgroundColor: UIColor = attributes.backgroundColor
+        
+        label.text = text
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.sizeToFit()
+        label.backgroundColor = backgroundColor
+        
+        if BaseTractElement.isParagraphElement(self) {
```

Comment:
most of lines here are the same, except for the style. Can you just set the style in the if/else block, and write the other lines just once.